gqlalchemy/models.py

```diff
@@ -149,8 +159,10 @@ class MemgraphTrigger:
     def to_cypher(self) -> str:
         """Converts a Trigger to a cypher clause."""
         query = f"CREATE TRIGGER {self.name} "
-        # when self.event_object is TriggerEventObject.ALL there is a double space
-        query += f"ON {self.event_object} {self.event_type} "
+        if self.event_object is TriggerEventObject.ALL:
```

Comment:
"ON " part of the string could have been added to query before the branching. 

Also, can we move this branching into the string?